db/migrate/20170103085726_create_cashboxes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCashboxes < ActiveRecord::Migration
+  def change
+    create_table :cashboxes do |t|
+      t.integer :balance, default: 0
```

Comment:
For the record, I'm still not convinced we should store balance at all.  It is calculable.  This seems like a premature scaling optimisation.  If everyone else agrees that we should store it in the db, then we need to ensure only the model itself can write to it, eg `def balance=; raise; end`  When and if the time comes where this is causing db thrashing then we can address it separately YAGNI